rundeckapp/src/main/groovy/org/rundeck/jaas/jetty/JettyAuthPropertyFileLoginModule.java

```diff
@@ -48,7 +48,8 @@ public void initialize(Subject subject, CallbackHandler callbackHandler, Map sha
             logger.debug("using reloadable realm property file reader");
             module = new ReloadablePropertyFileLoginModule();
         } else {
-            module = new PropertyFileLoginModule();
+            module = new ReloadablePropertyFileLoginModule();
```

Comment:
why not just simplify the whole if clause?

Reply:
Nice catch.